browser/js/common/directives/navbar/navbar.html

```diff
@@ -106,9 +98,9 @@
 	<div ng-class='class'>
 		<div class='dropdown-lists'>
 
-			<div class='dropdown'>
-				<div class='dropdown-title' ng-click='goToSection(".overview", "home")'>What is Kinetic</div>
-				<div class='dropdown-content'>
+			<div class='dropdown' ng-init='whatIs = true'>
+				<div class='dropdown-title' ng-click="whatIs = !whatIs; getInvolved = true; resources = true; ourTeam = true">What is Kinetic</div>
```

Comment:
You seem to be doing some version of this logic a bunch. Try abstracting it out into a function on the scope to avoid the repetition.